strongbox-storage/strongbox-storage-api/src/main/java/org/carlspring/strongbox/artifact/locator/handlers/AbstractArtifactLocationHandler.java

```diff
@@ -55,7 +55,7 @@ protected boolean isMetadata(Path path)
     {
         try
         {
-            return Boolean.TRUE.equals(RepositoryFiles.isMetadata((RepositoryPath) path));
+            return Boolean.TRUE.equals(RepositoryFiles.isLsm((RepositoryPath) path));
```

Comment:
@sbespalov : What is `isLsm(...)`? What doest this abbreviation stand for?

Reply:
it's Layout Specific Metadata, which probably can be better named.

Reply:
I think you should just call it `isLayoutSpecificMetadata()` in this case.

Reply:
ok, I just thought that such name will be too long :)

Reply:
@carlspring, while working with this PR I realized that we don't need such attribute. There will be enough just `isArtifact` and `isMetada` attributes, which can be overriden in concrete `LayoutProvider` implementation. For example in case of maven we will have following:
- `isMetadata = .pom || maven-metadata.xml`
- `isArtifact = !maven-metadata.xml && !index && !checksum && !diractory`

So we will store `ArtifactEntry` in case of just `isArtifact`, and for example we can detect `maven-metadata.xml` with `!isArtifact && isMetadata`.